x/evm/keeper/grpc_query_test.go

```diff
@@ -47,3 +47,47 @@ func TestQueryPointer(t *testing.T) {
 	require.Nil(t, err)
 	require.Equal(t, types.QueryPointerResponse{Pointer: seiAddr5.String(), Version: uint32(erc721.CurrentVersion), Exists: true}, *res)
 }
+
+func TestQueryPointee(t *testing.T) {
```

Comment:
can we also test for invalid / not registered yet?

Reply:
good point will add it